packages/node_modules/@ciscospark/common/src/constants.js

```diff
@@ -1,49 +1,55 @@
-export const WEBEX_EVENT = {
-  TEAMS_ACTIVITY: 'event:conversation.activity'
-};
-
-export const WEBEX_SPACE_TYPE = {
-  DIRECT: 'direct',
-  GROUP: 'group'
-};
-
-export const API_ACTIVITY_FIELD = {
-  ACTOR: 'actor',
-  OBJECT: 'object',
-  TARGET: 'target'
-};
-
-export const API_ACTIVITY_VERB = {
-  ACKNOWLEDGE: 'acknowledge',
-  POST: 'post',
-  SHARE: 'share',
-  ADD: 'add',
-  LEAVE: 'leave',
-  ADD_MODERATOR: 'assignModerator',
-  REMOVE_MODERATOR: 'unassignModerator',
-  HIDE: 'hide'
-};
-
 export const SDK_EVENT = {
-  CREATED: 'created',
-  DELETED: 'deleted',
-  UPDATED: 'updated',
-  SEEN: 'seen'
-};
-
-export const SDK_EVENT_OWNER = {
-  CREATOR: 'creator',
-  ORG: 'org'
-};
-
-export const SDK_EVENT_STATUS = {
-  ACTIVE: 'active',
-  DISABLED: 'disabled'
-};
-
-export const SDK_EVENT_RESOURCE = {
-  MEMBERSHIPS: 'memberships',
-  MESSAGES: 'messages'
+  FROM_WEBEX: {
+    TEAMS_ACTIVITY: 'event:conversation.activity',
+    ACTIVITY_FIELD: {
+      ACTOR: 'actor',
+      OBJECT: 'object',
+      TARGET: 'target'
+    },
+    ACTIVITY_VERB: {
+      ACKNOWLEDGE: 'acknowledge',
+      CREATE: 'create',
+      POST: 'post',
+      SHARE: 'share',
+      DELETE: 'delete',
+      ADD: 'add',
+      LEAVE: 'leave',
+      ADD_MODERATOR: 'assignModerator',
+      REMOVE_MODERATOR: 'unassignModerator',
+      HIDE: 'hide',
+      UPDATE: 'update'
+    },
+    ACTIVITY_TAG: {
+      HIDDEN: 'HIDDEN',
+      ONE_ON_ONE: 'ONE_ON_ONE',
+      LOCKED: 'LOCKED'
+    }
+  },
+  TO_APP: {
```

Comment:
And here, `EXTERNAL`.

Reply:
Agree, see commit 41ff1a54e925f04e1588abc4d53b969c05b123cc